examples/Demo/Shared/Pages/DataGrid/Examples/DataGridVirtualize.razor

```diff
@@ -73,5 +73,6 @@
         await Task.Delay(1500);
 
         items = GenerateSampleGridData(5000);
+        grid?.SetLoadingState(false);
```

Comment:
previously the assumption was that the grid would set loading to false - since it's being controlled, that's not accurate